ExoCTK/ldc/ldcfit.py

```diff
@@ -10,7 +10,6 @@
 import matplotlib.pyplot as plt
 from matplotlib import rc
 from scipy.optimize import curve_fit
-from scipy.interpolate import RegularGridInterpolator
 try:
```

Comment:
Remove the try/except statements. We want to always use relative imports within the package.  `from ExoCTK import` only works if you have a previous version installed and doesn't solve the issue.